tests/tasks/airbyte/integration_test_airbyte.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from prefect.engine.signals import FAIL
+from prefect.tasks.airbyte import AirbyteConnectionTask
+from prefect.tasks.airbyte.airbyte import ConnectionNotFoundException
+
+# TODO - Missing test for Connection with existing Schedule
+# i.e. Check that Schedule is removed and replaced with manual sync ...
+class TestAirbyteIntegration:
+
+    """
+    Integration tests for AirbyteConnectionTask
+
+    Please ensure Airbyte Server is running (locally) ...
+
+    """
+
+    # The Connection id to use in integration tests
+    CONNECTION_ID = "749c19dc-4f97-4f30-bb0f-126e53506960"
+
+    @pytest.mark.skip(reason="integration test - requires running Airbyte Server ...")
```

Comment:
Not sure if these integration tests belong here.. the code would have to be edited for them to run?

Reply:
These tests were used to verify functionality against a running Airbyte server. Since they require a running Airbyte server that has been specifically configured, I've removed them from this PR.